src/main/java/core/basesyntax/PrintInfo.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface PrintInfo {
+    void printInfo(Figure figure);
```

Comment:
1. better name suggestion
2. same, dont need parameter. refer to my comment at the AreaCalculator interface
```suggestion
    void draw();
```